### PR TITLE
Bug OCPBUGS-2886: Add note about unnecessary load-balancer setting on CP

### PR DESCRIPTION
**Pull request by MaysaMacedo**
When using Kuryr the OpenShift Services are already handled by Kuryr consequently Octavia and not by the cloud provider. The changes the user might attempt to do in the LoadBalancer section will not have any effect in the load-balancers created by Kuryr. This commit fixes the issue by stating that config section is only valid for installations without Kuryr and the reasoning behind it.

Version(s):

All openshift versions that cloud provider is supported

Issue: OCPBUGS-2886

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by MaysaMacedo** [comment]
@maxwelldb It seems that doc is also used in installations without Kuryr, let me know if this format looks okay to you or you would prefer a separate cloud-provider-setting docs only for Kuryr.

File: modules/installation-osp-setting-cloud-provider-options.adoc [hunk 1]
```diff
@@ -64,6 +64,11 @@ Prior to saving your changes, verify that the file is structured correctly. Clus
 ====
 You must set the value of the `create-monitor` property to `True` if you use services that have the value of the `.spec.endpointTrafficPolicy` property set to `Local`. The OVN Octavia provider in {rh-openstack} 16.1 and 16.2 does not support health monitors. Therefore, services that have `ETP` parameter values set to `Local` might not respond when the `lb-provider` value is set to `"ovn"`.
 ====
++
+[IMPORTANT]
+====
+For installations with Kuryr, the Services are handled by Kuryr, consequently there is no need to configure octavia load balancing in the cloud provider.
```
Comment (maxwelldb) [inline modules/installation-osp-setting-cloud-provider-options.adoc:70]:
```suggestion
For installations that use Kuryr, Kuryr handles relevant services. There is no need to configure Octavia load balancing in the cloud provider.
```

File: modules/installation-osp-setting-cloud-provider-options.adoc [hunk 2]
```diff
@@ -31,7 +31,7 @@ $ vi openshift/manifests/cloud-provider-config.yaml
 
 . Modify the options based on link:https://v1-18.docs.kubernetes.io/docs/concepts/cluster-administration/cloud-providers/#cloud-conf[the `cloud.conf` specification].
 +
-Configuring Octavia for load balancing is a common case. For example:
+Configuring Octavia for load balancing is a common case for installations that do not use Kuryr. For example:
```
Comment (maxwelldb) [inline modules/installation-osp-setting-cloud-provider-options.adoc:34]: @MaysaMacedo Actually, could one of these work better?

"installations" -> "clusters"
"installations" -> "deployments" 

After that, I think this just needs a QE ack.

Comment (MaysaMacedo) [reply modules/installation-osp-setting-cloud-provider-options.adoc:34]: clusters I think is better. Let me update it

**Comment by rlobillo** [comment]
Hello @maxwelldb!

I think we can address on this file, probably on different PR, the resolution of these doc bugs:

https://issues.redhat.com/browse/OCPBUGS-2350
https://issues.redhat.com/browse/OCPBUGS-2217
https://issues.redhat.com/browse/OCPBUGS-2789

Thanks!

**Comment by MaysaMacedo** [comment]
This PR seems ready. @maxwelldb can you have another look? Thanks

**Comment by maxwelldb** [comment]
Looks fine to me.

@MaysaMacedo Is this 4.12+, or 4.13+?

**Review by maxwelldb — APPROVED** [review]

**Comment by MaysaMacedo** [comment]
@maxwelldb https://github.com/openshift/openshift-docs/pull/52493

**Pull request merged**